src/views/preview/preview.scss

```diff
@@ -469,8 +469,40 @@ $stage-width: 480px;
             display: flex;
             flex: 1;
         }
+
+        ::-webkit-input-placeholder { /* Chrome <=56, Safari < 10 */
+            transition: opacity .25s;
+            opacity: 1;
```

Comment:
The "preview page" is the same as the 3.0 "project page". It's confusing because for a long time we called it the "preview page" to make clear that the regular, 2.0 project page was the standard one, and this 3.0 project page was under construction. Now that it's basically ready for launch, we've made regular project page urls (like `/projects/<project id here>`) point to it. (But they won't *actually* point to it until we push `www` to the `master` branch on Jan 2!)

As for which browsers, I think we generally try to make our css support all the officially supported browsers. Here, I'm supporting a few more versions, because it's easier to do that than to pick out precisely which ones we don't need to support! (Normally, CSS doesn't need to be this picky, but this feature has somewhat shaky support across browsers.)

Does that make it clearer?